cli/src/semgrep/cli.py

```diff
@@ -10,17 +10,25 @@
 from semgrep.commands.lsp import lsp
 from semgrep.commands.publish import publish
 from semgrep.commands.scan import scan
+from semgrep.constants import DEFAULT_EPILOGUE
 from semgrep.default_group import DefaultGroup
 from semgrep.state import get_state
 from semgrep.verbose_logging import getLogger
 
 logger = getLogger(__name__)
 
 
-@click.group(cls=DefaultGroup, default_command="scan", name="semgrep")
+@click.group(
+    cls=DefaultGroup, default_command="scan", name="semgrep", epilog=DEFAULT_EPILOGUE
+)
 @click.help_option("--help", "-h")
 @click.pass_context
 def cli(ctx: click.Context) -> None:
+    """
```

Comment:
This became useless to change because this is not used anymore. Semgrep --help is now handled by osemgrep so this part of pysemgrep is dead.

Reply:
@aryx – Given that we're writing the help message manually for `osemgrep`, we could use the click to write the help message output/
  https://github.com/returntocorp/semgrep/blob/a0033e754c59573b6b9a38b5ac7c2373d8854eb6/src/osemgrep/cli/CLI.ml#L25C48-L26C39

Reply:
the toplevel help message is handled manually, but the semgrep xxx --help for each subcommand is generated automatically by Cmdliner (the equivalent of Click but for OCaml), and I want the  source of truth to be osemgrep, not pysemgrep. In fact this week I'll remove all the help message for each CLI args in pysemgrep because it's redundant with the CLI parsing in osemgrep.

Reply:
for clarity, this line here is only used for the toplevel help message